tests/unit/controllers/join-test.js

```diff
@@ -0,0 +1,16 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'website-www/tests/helpers';
+
+module('Unit | Controller | join', function (hooks) {
```

Comment:
- Can you write a test for checking what is expected to render with step query param change?

- Also write a test for validating your logic for the rendering step, when using `localstorage` and other etc...

Reply:
We skip the test for the stepper component and for `join` controller see main comment(updated).

Reply:
It's not for the stepper component, it's like what you expect when step query param changes.